CHANGELOG.md

```diff
@@ -3,7 +3,13 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
  [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
-## [0.1.5] - 2022-07-11
+## [0.2.0] - 2022-10-28
+
++ Add - New schema `ephys_report` to compute and store figures from results
++ Add - Widget to display figures
++ Add - Add `ephys_no_curation` and routines to trigger spike-sorting analysis using Kilosort (2.0, 2.5)
+
+## [0.1.5] - 2022-09-29
```

Comment:
```suggestion
## 0.1.5 - 2022-09-29
```

Reply:
Removing the link since we haven't made a release for `0.1.5`.  Alternatively, we could add a link that compares version `0.1.5` to `0.1.4`.